src/Commands/Env/CloneContentCommand.php

```diff
@@ -0,0 +1,26 @@
+<?php
+namespace Pantheon\Terminus\Commands\Env;
+
+use Pantheon\Terminus\Commands\CloneCommand;
+
+class CloneContentCommand extends CloneCommand
+{
+
+    /**
+     * Clones content from one environment to another.
+     *
+     * @command env:clone-content
+     *
+     * @usage terminus env:clone foo.live foo.dev
+     *   Clones the files and database from the live environment of the 'foo'
+     *   site to the development environment of the 'foo' site.
+     *
+     * @param $origin
+     * @param $target
+     *
+     */
+    public function cloneContent($origin, $target)
+    {
+        $this->invokeClone($origin, $target, ['database', 'files']);
```

Comment:
To also chime in, I would like to avoid abstracting command logic at this time. I'd rather see duplicate code and a better understanding of the new command system once we have more experience under our belts.